lib/expression/v2/callbacks.ex

```diff
@@ -54,10 +54,17 @@ defmodule Expression.V2.Callbacks do
   def callback(module \\ Standard, context, function_name, arguments) do
     case implements(module, function_name, arguments) do
       {:exact, module, function_name, _arity} ->
```

Comment:
just curiousity: is this `implements/3` function used anywhere else? I see we are not using `arity` at all

Reply:
No, afaik it's only used here? I copied it from your original implementation :)

Reply:
I think it has always returned the `arity` but we are not using it anywhere. Should we just refactor it?

Reply:
yeah feel free to do so in a separate PR ?

Reply:
actually I still have some stuff to do here, I'll include it here.